packages/core/src/Modules/Cashier/Containers/on-ramp.jsx

```diff
@@ -16,9 +28,13 @@ const OnRamp = ({
     resetPopup,
     setIsOnRampModalOpen,
     should_show_dialog,
+    setSideNotes,
 }) => {
     React.useEffect(() => {
         onMountOnramp();
+
+        setSideNotes([<OnRampSideNote key={0} />]);
```

Comment:
`setSideNotes` will be passed only on Desktop. I think we should a condition to check if `setSideNotes` is defined so that it doesn't break on mobile

Reply:
@mustofa-binary , please make the changes to only have `setSideNotes` on `desktop`.